### ISSUE TITLE
Hide not complete features behind 'beta' tag

### ISSUE THREAD

**Issue by MarkSummerville** [TV-Rename/tvrename#277]
Idea is to disable:
 * Actors Grid (too slow)
 * 2x uTorrent functions (not really sure how they work)
* ShowSummary
 * Duplicate Identifier

and only show them if the user is in 'beta' mode.

This way they can stay in the codebase, but not confuse users

**Comment by MarkSummerville** [comment]
Idea generated from #262

**Comment by Andy-ABTec** [comment]
I can easily hide the Actors Grid and the µTorrent functions in the docs when you're ready, but not sure what you mean by Show Summary and Duplicate Identifier - anything I need to do for those?

**Comment by Andy-ABTec** [comment]
Cool, I was just checking if I knew either of these by a different name, but I don't, they're not in the docs so no problem... Everything mentioned earlier will be "gone" in my next push

Will you be launching a new release to hide the torrent and actor grid stuff? (would be useful to test the new download link...)

**Comment by MarkSummerville** [comment]
Yes, will be a new release, but it’s not coded up yet, so could be a couple
of weeks away

**Comment by Andy-ABTec** [comment]
Might be worth looking at hiding Add/Edit  Search tab as well...

**Comment by MarkSummerville** [comment]
I think that works enough to keep at the moment. I may try and fix any quick bugs with it though
